tests/kata-containerized-test-tool/Makefile

```
@@ -0,0 +1,16 @@
+BINARY_NAME=kata-containerized-test-tool
+
+.PHONY: build
+build:
+	go build -o $(BINARY_NAME) cmd/katatest/main.go
+
+.PHONY: docker
+docker: build
+	docker build -t kata-test-container .
+
+.PHONY: clean
+clean:
+	rm -f $(BINARY_NAME)
+
+.PHONY: all
+all: clean build docker
```

We shouldn't automatically clean, in case we can just reuse the binary and save time - go will check that for us.

```suggestion
all: build docker
```